src/components/FindMovie/FindMovie.tsx

```diff
@@ -1,54 +1,128 @@
-import React from 'react';
+import React, { useCallback, useState } from 'react';
+import classnames from 'classnames';
 import './FindMovie.scss';
 
-// import { MovieCard } from '../MovieCard';
-
-export const FindMovie: React.FC = () => (
-  <>
-    <form className="find-movie">
-      <div className="field">
-        <label className="label" htmlFor="movie-title">
-          Movie title
-        </label>
-
-        <div className="control">
-          <input
-            type="text"
-            id="movie-title"
-            placeholder="Enter a title to search"
-            className="input is-danger"
-          />
-        </div>
+import { MovieCard } from '../MovieCard';
 
-        <p className="help is-danger">
-          Can&apos;t find a movie with such a title
-        </p>
-      </div>
+interface FindMovieProps {
+  onAddMovie: (newMovie: Movie | null) => void;
+}
+
+export const FindMovie: React.FC<FindMovieProps> = ({ onAddMovie }) => {
+  const [movie, setMovie] = useState<Movie | null>(null);
+  const [title, setTitle] = useState('');
+  const [noMovieError, setNoMovieError] = useState('');
```

Comment:
This variable implies that the value is boolean but it is not

Reply:
Have changed thanx